src/vit-servicing-station/vit-servicing-station-lib/src/db/queries/funds.rs

```diff
@@ -28,23 +28,34 @@ fn join_fund(
     fund.chain_vote_plans = voteplans_dsl::voteplans
         .filter(voteplans_dsl::fund_id.eq(id))
         .load::<Voteplan>(db_conn)
-        .map_err(|_e| HandleError::NotFound("Error loading voteplans".to_string()))?;
+        .map_err(|e| {
+            HandleError::NotFound(format!("Error loading voteplans, error: {}", e.to_string()))
+        })?;
```

Comment:
This file will conflict with some changes I made to support both Postgres and SQLite databases. I'm happy to help resolve it if you like :)